.buildkite/scripts/steps/functional/scalability_dataset_extraction.sh

```diff
@@ -33,11 +33,11 @@ mkdir "${BUILD_ID}"
 # Archive json files with traces and upload as build artifacts
 tar -czf "${BUILD_ID}/scalability_traces.tar.gz" -C target scalability_traces
 buildkite-agent artifact upload "${BUILD_ID}/scalability_traces.tar.gz"
-# Upload Kibana build, plugins, commit sha and traces to the bucket
-download_artifact kibana-default.tar.gz ./"${BUILD_ID}"
-download_artifact kibana-default-plugins.tar.gz ./"${BUILD_ID}"
```

Comment:
@spalger I was doing it on purpose: scalability [pipeline](https://buildkite.com/elastic/kibana-scalability-benchmarking) uses the latest successful run of single user performance run, specifically kibana build, plugins and json files collected during that run. We want to run scalability on the very same build and using existing one saves time.

We upload them in [gcs bucket](https://console.cloud.google.com/storage/browser/kibana-performance) for each successful run.

Maybe uploading build and plugin is redundant and I can fetch them from buildkite artifacts storage directly?

Reply:
Ah, I missed that the download location is within the `BUILD_ID dir

Reply:
Fixed and trying out in https://buildkite.com/elastic/kibana-single-user-performance/builds/3300